src/util.jl

```diff
@@ -124,6 +124,21 @@ function eatnewlines(str, i=1, l=lastindex(str))
     return i, count
 end
 
+# Move past consecutive lines that start with commentchar.
+# Return a tuple of the new pos in str and the amount of comment lines moved past.
+function eatcommentlines(str, i=1, l=lastindex(str), commentchar::Union{Char, Nothing}=nothing) 
+    commentchar == nothing && return i, 0
```

Comment:
compare like `commentchar === nothing` to make this even easier on the compiler

Reply:
Thanks for pointing this out. I have made this change in 9b1a8cd40cced37a3655cb50575593fca40f1ba4